internal/handlers/types.go

```diff
@@ -42,14 +42,13 @@ type signDuoRequestBody struct {
 	TargetURL string `json:"targetURL"`
 }
 
-// firstFactorBody represents the JSON body received by the endpoint.
+// firstFactorRequestBody represents the JSON body received by the endpoint.
 type firstFactorRequestBody struct {
-	Username  string `json:"username" valid:"required"`
-	Password  string `json:"password" valid:"required"`
-	TargetURL string `json:"targetURL"`
-	// Cannot require this field because of https://github.com/asaskevich/govalidator/pull/329
-	// TODO(c.michaud): add required validation once the above PR is merged.
-	KeepMeLoggedIn *bool `json:"keepMeLoggedIn"`
+	Username       string `json:"username" valid:"required"`
+	Password       string `json:"password" valid:"required"`
+	TargetURL      string `json:"targetURL"`
+	RequestMethod  string `json:"requestMethod"`
+	KeepMeLoggedIn *bool  `json:"keepMeLoggedIn"`
```

Comment:
```suggestion
	KeepMeLoggedIn *bool  `json:"keepMeLoggedIn" valid:"required"`
```

I believe this should now be okay, also if changed you need to modify https://github.com/authelia/authelia/blob/1733762f681447b3adb55ba5528a8ce09952c3c0/api/openapi.yml#L506-L508.

Reply:
I attempted to make this change, it failed but didn't spend time to figure out why.